src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -686,9 +686,17 @@
                 "nexthop_group_threshold_type": "percentage",
                 "polling_interval": "0"
             }
+        },
+
+        "DSCP_TO_TC_MAP": {
+	        "Dscp_to_tc_map1": { 
+		        "24": "3", 
```

Comment:
This representation is different than other lists, I think in future we should change it as below, so that the translation code is not burdened by special cases:    @lguohan  @renukamanavalan for comment.
```
"DSCP_TO_TC_MAP": {
    "Dscp_to_tc_map1": { 
           "24": {
               "tc": "3"
           }
      }
 }

Reply:
what does the redis entry looks like?

Reply:
It looks like exactly like line 693:  "24": "3", in which "24" is the field name, "3"  is the value.